CharacterCreation/03_Feats.txt

```diff
@@ -545,29 +545,28 @@ Lucky:
 Prerequisites: Luck 6
 	Good things predictably happen to you.
 	Luck = 6	
-		You can choose to reroll a die, once per long rest. You can only reroll 
-		once per separate die use and if you reroll, you must keep the second roll.
+		You can choose to reroll a check, save, or attack once per long rest. You 
+		may only reroll once for a given action and you must keep the second result.
 	Luck = 7		
-		You can choose to reroll a die, twice per long rest. You can only reroll
-		once per separate die use and if you reroll, you must keep the second roll.
+		You can choose to reroll a check, save, or attack twice per long rest. You 
+		may only reroll once for a given action and you must keep the second result.
 	Luck = 8		
-		You can choose to reroll a die, twice per long rest. You can only reroll 
-		once per separate die use and if you reroll, choose which roll to keep.
+		You can choose to roll a check, save, or attack with advantage twice per 
+		long rest.
 	Luck = 9		
-		You can choose to reroll a die, three times per long rest. You can only
-		reroll once per separate die use and if you reroll, choose which roll to keep.
+		You can choose to roll a check, save, or attack with advantage three times 
+		per long rest.
 	Luck = 10		
-		You can choose to reroll a die, four times per long rest. You can reroll
-		as many times as you want (which uses up your rerolls-per-rest) and if you 
-		reroll, choose which of the rolls to keep.
+		You may choose to reroll a check, save, or attack four times per long rest. 

```

Comment:
Reroll means you must take the second roll regardless of whether the second roll is superior or not. I don't see the need to formally define reroll, since it's a word in the English language.

Reply:
The problem with using Advantage specifically in this spot is that in that, with advantage terminology, this subfeat is allowing you to choose whether to make a roll with no advantage, advantage, double advantage, or triple advantage, and your choice makes a difference to how many times you can do it again. The original phrasing is more clear.

Reply:
@bleehu the wording here is supposed to be saying a fancy kind of "rolling with advantage"... unless you meant to clarify something else?

@Turtlelord26 , thanks, that makes sense.

Reply:
@1sourcecontrol 
Reroll:
take the 2nd roll
Advantage:
take the higher roll